packages/react-img-viewer/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "version": "0.1.0",
```

Comment:
**Chore**
We need to define the library name. Also let's keep the lib name as `react-iv-viewer` as core library is called iv-viewer and this is port of it on react.
```
"name": "react-iv-viewer"
```

Reply:
Same need to be changed on folder name as well from `react-img-viewer` to `react-iv-viewer` everywhere.